tagcompare/capture.py

```diff
@@ -175,7 +225,8 @@ def main():
     output.aggregate()
 
     configs = settings.DEFAULT.configs_in_comparisons()
-    __capture_tags_for_configs(cids=cids, pathbuilder=pathbuilder, configs=configs)
+    _capture_tags_for_configs(
+        cids=cids, pathbuilder=pathbuilder, configs=configs)
```

Comment:
The original build is used when we chain capture.py with compare.py, which used to be the main scenario for tagcompare.  In this case they need to share the same build to do compare against and we aren't returning this as a means to indicate error or not (see https://github.com/paperg/tagcompare/blob/master/tagcompare/main.py#L110-L112).